tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerCoverageSuite.scala

```diff
@@ -0,0 +1,114 @@
+package scala.meta.tests.tokenizers
+
+import scala.meta._
+
+class TokenizerCoverageSuite() extends BaseTokenizerCoverageSuite {
+  
+  // Term
+  check[Term.Annotate]("→(a)←: →@A←")
+  check[Term.Apply]("→(f)←(→(((a)))←)")
+  check[Term.Apply]("→(f)←(→(a)←)")
+  check[Term.Apply]("→(f)←(→{ case a => a }←)")
+  check[Term.Apply]("→(f)←(→{ x }←)")
+  check[Term.ApplyInfix]("→(a)← →op← →(b)←")
+  check[Term.ApplyInfix]("→(a, b)← →op← (→c←, →d←)")
+  check[Term.ApplyInfix]("→(a, b)← →op← →((c, d))←")
+  check[Term.ApplyInfix]("→1← →+← →1←")
+  check[Term.ApplyInfix]("→a← →f← →(b)←")
+  check[Term.ApplyType]("→(f)← [→A←,→B←]")
+  check[Term.ApplyType]("→(f)← [→A←]")
+  check[Term.ApplyUnary]("→-← →(a)←")
+  check[Term.Ascribe]("→(a)←: →(A)←")
+  check[Term.Assign]("→(a)← = →(b)←")
+  check[Term.Block]("{ →(a)←; →(b)← }")
```

Comment:
this does not capture the semicolon

Reply:
Semicolon is considered to be a statement separator, just like newline. I think it makes sense for it to not be part of any statements, just like newline.